vitrina/structure/models.py

```diff
@@ -380,6 +386,12 @@ def description(self):
             return metadata.description
         return ""
 
+    @builtins.property
```

Comment:
One method here (in this file) is decorated with `@property` the other with `@builtins.property`. Would be nice to keep consistent if there is no particular reason for the difference in usage